ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthCall.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
This file is unrelated to the aims of this PR.  Please remove.  If it is needed to demonstrate Sonar Deltas then it has been proven and can be removed.

Reply:
Yep, none of these should be in here, remnants of trying to debug the concurrency issue when debugging tests. Will remove.